### ISSUE TITLE
Feature request: Video in iframe in HTML but excluded from PDF

### ISSUE THREAD

**Issue by albertdev** [dotnet/docfx#4870]
Like the subject says, I want to include a video (shown in an iframe) on the HTML version of the generated website. The page is generated from a Markdown file.

It's possible to add the video by including the following line in the Markdown file, and it will work just fine:

```
<iframe src="https://player.vimeo.com/video/111111111" width="640" height="360" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
```

The problem comes when I want to generate PDF files from those same Markdown files. It looks like wkhtmltopdf does its best to download something, and I believe the `Exit with code 1 due to network error: ContentNotFoundError` it spits out might be an indication that it failed.

At the end of the process my PDF ends up containing whitespace where the video would be. I would rather make an extra link in the text before the video then having that whitespace there.

On a related note: it seems DocFX completely ignores the errors wkhtmltopdf generates, they don't even show up as warnings at the end of the DocFX process. Is there a reason for that?

**Comment by albertdev** [comment]
Please note that I have tested the `[!Video https://...]` tag in DocFX 2.43.2 but it doesn't seem to be working; the tag shows up in the output without any interpretation.

**Comment by donhuvy** [comment]
Can I use Vimeo/YouTube video embed without touch in generated HTML file? I want embed video from markdown (*.md) ?

**Comment by superyyrrzz** [comment]
To the last question, wkhtmltopdf sometimes wrote not fatal error into stderr, which unexpected fails docfx. So finally, docfx chooses not to fail if wkhtmltopdf fails in #4733.

**Comment by yufeih** [comment]
Addressed in v2.73.0 with [a new PDF engine](https://dotnet.github.io/docfx/docs/pdf.html).
